### ISSUE TITLE
[Bug]: 'Select' Element does not apply custom theme color to label and no way to differentiate between label text and control text in properties

### ISSUE THREAD

**Issue by placidic** [lowcoder-org/lowcoder#812]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Currently, theme colors defined within a custom theme do not get applied to the Select Component Element.
Theme preview showing no update to the Select label

![Screenshot 2024-04-17 083944](https://github.com/lowcoder-org/lowcoder/assets/1841459/9c7b48cf-2cd9-4806-9182-d75408c97a93)
![Screenshot 2024-04-17 084016](https://github.com/lowcoder-org/lowcoder/assets/1841459/629a52a9-5b16-4064-945b-a17ff3c09303)

### Expected Behavior

It is expected that the label element of the Select component, matches the predefined text color defined in the custom theme.

### Steps to reproduce

1. Create a Theme with custom text color.
2. Preview Theme and observe preview

### Environment

app.lowcoder.cloud - Verwion 2.3.1

**Comment by placidic** [comment]
We've also tried a few things like using the CSS scripts to modify the text label, which oddly worked while in edit mode, but when you would preview or view the app, the updated colors would disappear and revert back to the incorrect color.

There is also no current way to differentiate between colors within the label vs the control (like the text inside the dropdown) - might be a feature to look to add a specific label color property to controls for more flexibility.

**Comment by placidic** [comment]
Unfortunately, It appears that the 2.4 Update has created new issues with the default components. Please see the screenshot below for the v2.4 output of the same settings:
![image](https://github.com/lowcoder-org/lowcoder/assets/1841459/9a267cc6-fd3c-472e-9442-f36b089f9c9b)

**Comment by FalkWolsky** [comment]
Yes. We addressed it already. It should be fixed in a good way in v2.4.1.
We made a style "reset" in 2.4.0 - which means to clean up the standard values after all changes. Also in the past days, we made sure, that the values of themes are used proper. Please give us some bit more time and with the release it should be ok then.
